packages/api/start/routes/technologyOrdersChat.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable jsdoc/check-tag-names */
+/* eslint-disable jsdoc/check-indentation */
+const Route = use('Route');
+
+Route.get(
+	'technologies/:technology([0-9]+)/orders/:order([0-9]+)/chat',
```

Comment:
We don't need this number range
```suggestion
	'technologies/:technology/orders/:order/chat',
```

Reply:
why not? this will force to only match routes with the ids as numbers, the following `technologies/new/orders/create/chat` will not match with my approach, if I remove the regex, the route will match with my example with sometime is not what we want

Reply:
Ok, it's fine. I don't think it's worth the effort, but it can continue, so 🤔